src/base/RefCount.h

```diff
@@ -27,6 +27,10 @@ class RefCount
 {
 
 public:
+    template<typename... Args>
+    inline static auto Make(Args&&... args) {
+        return RefCount<C>(new C(std::forward<Args>(args)...));
```

Comment:
`std::forward` is declared in [utility](https://en.cppreference.com/w/cpp/header/utility). We should `#include` that header explicitly instead of waiting for that inclusion to become necessary.